FFmpegInterop/CodecChecker.h

```diff
@@ -376,14 +400,36 @@ namespace FFmpegInterop
 				if (codecId == AV_CODEC_ID_MPEG2VIDEO)
 				{
 					result &= CheckIsMpeg2VideoExtensionInstalled();
+					if (!result && !hasAskedInstallMpeg2Extension)
+					{
+						RaiseCodecRequired(CodecRequiredReason::HardwareAcceleration, "MPEG2", "MPEG2 Video Extension", "9n95q1zzpmh4");
```

Comment:
Maybe it would be better to inject these IDs as static configuration values instead of hard-coding them?

Reply:
There are only 4 video extensions available in the Store, and all of them are from Microsoft. As long as there is not a single 3rd party alternative in the Store, I do not see a need to make this configurable...